bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/dto/clip2/LightLevelReport.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.hue.internal.dto.clip2;
+
+import java.time.Instant;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * DTO for CLIP 2 light level sensor report.
+ *
+ * @author Jacob Laursen - Initial contribution
+ */
+@NonNullByDefault
+public class LightLevelReport {
```

Comment:
Furthermore, we need to discuss who will handle the following tasks for the full range of sensor types..
1. Complete of the PR for Motion, Temperature, LightLevel => @jlaur => keep scope as is, or include 2. as well?
2. Make the same analogous changes for existing Button and RelativeRotary sensors => @jlaur include in this PR, or  @andrewfg create a new PR?
3. Implement the new home security sensors, (including the same analogous code) as above => @andrewfg

Reply:
Apropos `camera_motion` .. good news: your code in this PR will work for a camera exactly the same as for a PIR motion detector; so we can re-use the same channels for it (but I may need to modify the readme and the thing xml to clarify that the motion may be triggered either by a PIR or a camera).

Reply:
I don't think inheritance for DTO classes would be a good design choice. It introduces complexity and tight coupling between the base class and subclasses. I would generally favor composition. However, in this case they share only one field. In some cases even zero, as you pointed out (`changed` vs. `updated`).

Therefore I would leave these few lines of "duplication" since these DTO are similar, but not identical.